.github/workflows/all.yml

```diff
@@ -43,27 +43,17 @@ jobs:
 
   lint:
     runs-on: ubuntu-latest
-    env:
-        GO111MODULE: 'on'
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Install Go
-        uses: actions/setup-go@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
         with:
-          go-version: 1.17.x
-      - name: Install golangci-lint
-        working-directory: /tmp
-        run: |
-          GOLANGCI_LINT_VERSION=$(head -n 1 "${GITHUB_WORKSPACE}/.golangci.yml" | tr -d '# ')
-          go install "github.com/golangci/golangci-lint/cmd/golangci-lint@$GOLANGCI_LINT_VERSION"
-      - name: Run linters
-        run: |
-          BASEREV=$(git merge-base HEAD origin/master)
-          echo "Base revision: $BASEREV"
-          golangci-lint run --out-format=tab --new-from-rev "$BASEREV" ./...
+          # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
+          version: v1.43
+          only-new-issues: true
```

Comment:
Ah, glad to see this works now. The reason we didn't use it before was that `only-new-issues` didn't work the same as `--new-from-rev` so it always showed old issues.